pull_request_template.md

```diff
@@ -0,0 +1,25 @@
+"What does this PR do?" This pull request created to get all form answers when
+the manger select the employee from the list to show result.And this happend by
+employee (id) joined with form (id) ,and questions (id) in the query.
+
+"Does this Fix an Issue?"
+ "Related #34"
+
+"How to test this PR"
+1-You should be able to access database on heroku by url.
+2-When you can access to database by url, implement db_build.sql by db_build.js
```

Comment:
He should NOT rebuild it, he needs to add the link in config.env file which he should create in root!

Reply:
I made brief description without rebuild .